config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 Please change value of `adapter` to `postgresql`, database adapter should be using `Postgresql` and add `encoding: unicode` after line `8` because this part should like this:

```ruby
# config/database.yml
...
default: &default
adapter: postgresql
encoding: unicode
...
```